packages/core/src/session_pool/session_pool.ts

```diff
@@ -290,6 +290,14 @@ export class SessionPool extends EventEmitter {
         return this._createSession();
     }
 
+    async resetStore() {
+        if (!this.keyValueStore) {
+            return;
+        }
+
+        await this.keyValueStore.setValue(this.persistStateKey, null);
```

Comment:
maybe this way its better?
```suggestion
        await this.keyValueStore?.setValue(this.persistStateKey, null);
```